Wrappers/Python/cil/optimisation/utilities/StepSizeMethods.py

```diff
@@ -82,6 +82,9 @@ class ArmijoStepSizeRule(StepSizeRule):
         The amount the step_size is reduced if the criterion is not met
     max_iterations: integer, optional, default is numpy.ceil (2 * numpy.log10(alpha) / numpy.log10(2))
         The maximum number of iterations to find a suitable step size 
+    warmstart: Boolean, default is True
+        If `warmstart = True` the initial step size at each Armijo iteration is the calculated step size from the last iteration. If `warmstart = False` at each  Armijo iteration, the initial step size is reset to the original, large `alpha`. 
```

Comment:
I don't follow. This implies to me:

```py
if not self.warmstart:
    self.alpha = self.alpha_orig
```

but the code says:

```py
if self.warmstart:
    self.alpha_orig = self.alpha
```

Reply:
Thanks Casper - i think they are doing the same thing but yours makes more sense!